airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -245,8 +245,9 @@ def display_logs_from(log_group: str, continuation_token: str | None) -> str | N
                 if e.response["Error"]["Code"] == "ResourceNotFoundException":
                     # we land here when the log groups/streams don't exist yet
                     self.log.warning(
-                        "No new Glue driver logs so far.\nIf this persists, check the CloudWatch dashboard "
-                        f"at: https://{self.conn_region_name}.console.aws.amazon.com/cloudwatch/home"
+                        "No new Glue driver logs so far.\n"
+                        "If this persists, check the CloudWatch dashboard at: %r.",
+                        f"https://{self.conn_region_name}.console.aws.amazon.com/cloudwatch/home",
```

Comment:
LGTM.  can we avoid f-string here?
"https://%s.console.aws.amazon.com/cloudwatch/home", self.conn_region_name

Reply:
Any reason for that? It is only arguments, not message itself.

In addition it only construct correct URI for default `aws` partition, but not for `aws-cn` and maybe not for us-gov and secrets/top-secrets partitions

Reply:
Yes. It's perfectly fine to use f-strings for warnings.

Reply:
Info/debug of course not (because of penalty you pay to process the f-string when your logging is disabled for that level, but warnings  you generally show always.